molecular/src/main/kotlin/com/hartwig/actin/molecular/MolecularInterpreterInputData.kt

```diff
@@ -101,12 +101,14 @@ object InputDataLoader {
                 }
             }
             val panelSpecifications = async {
+
                 withContext(Dispatchers.IO) {
                     LOGGER.info("Loading panel specifications from {}", config.panelSpecificationsFilePath)
-                    val panelSpecifications =
-                        config.panelSpecificationsFilePath?.let { PanelGeneSpecificationsFile.create(it) }
-                            ?: PanelSpecifications(emptyMap())
-                    panelSpecifications
+                    val serveRecord =
+                        deferredServeDatabase.await().records()[ServeLoader.toServeRefGenomeVersion(CLINICAL_TESTS_REF_GENOME_VERSION)]
+                    val knownGenes = serveRecord?.knownEvents()?.genes()?.toSet()?.map { it.gene() }?.toSet() ?: emptySet()
```

Comment:
I would've thought we had an easier way of doing this. You can probably skip the first `toSet()`:

```suggestion
                    val knownGenes = serveRecord?.knownEvents()?.genes()?.map { it.gene() }?.toSet().orEmpty()
```
Also, what does it mean for the serveRecord to be null? Should that be an error scenario? We probably don't want to run with an empty set of known genes.